swagger.yaml

```diff
@@ -35,6 +55,39 @@ responses:
     description: "Failed to process the request due to an internal error"
 
 definitions:
+  CacheTime:
+    type: object
+    required:
+      - id
+      - path
+      - etag
+    properties:
+      id:
+        $ref: "#/definitions/CacheTimeID"
+        description: "MD5 of the path"
+        type: string
+        example: "1b"
+      path:
+        description: "Path for which caching is set"
+        type: string
+        example: "admin"
+      etag:
+        description: "ETag for cache validation"
+        type: string
+        example: "etag_example"
+      collectionId:
+        description: "Collection ID"
```

Comment:
you can specify the length here with format: int32 or format: int64

Reply:
Do we know the length of our collectionId?

Reply:
I don't know, but we should be able to figure this out by checking Zebedee.

Reply:
question to ask ONS at some point. I assume this cache gets cleared in intervalls. Int32 still can handle 2,147,483,647 ids. That is probably more then mongodb collection can handle. [MongoDB recomend](https://www.mongodb.com/developer/products/mongodb/schema-design-anti-pattern-massive-number-collections/) a collection amount of 10000 -

Reply:
What is the difference between ID and collectionId in mongoDB? ID is the id string of one collection. So collectionId is another id just to reference one collection?

 I see one collection as one record when thinking about relational databases.

Reply:
In the context of mongoDB, our collection appears to be a table in the database, so I assume our `collectionId` will be a reference to a table and our `id` is a cachetime's unique identifier (which will be the md5 of the path)? Happy to be corrected though.